.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+sudo: true
+branches:
+  only:
+    - master
+    - /^\d\.\d+$/
+    - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
+matrix:
+  include:
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.6
+      env: TOXENV=docs
+before-install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libavahi-compat-libdnssd-dev
+install:
+  - pip install -U tox twine wheel codecov
+
+script: tox
+after_success:
```

Comment:
Is there a reason we don't do this always, not only on success?

Reply:
We can change it. I was building from a template.